src/mlpack/bindings/go/tests/gofile/go_binding_test.go

```diff
@@ -0,0 +1,351 @@
+package main
+
+import (
+	"mlpack/build/src/mlpack/bindings/go/mlpack"
+	"testing"
+
+	"gonum.org/v1/gonum/mat"
+)
+
+func TestRunBindingNoFlag(t *testing.T) {
+	t.Log("Test that when we run the binding correctly (with correct input parameters), we get the expected output.")
```

Comment:
Can you make sure the lines are 80 characters or less please?  (I guess the automatic style checker doesn't check .go files.)